packages/lib-classifier/src/plugins/drawingTools/models/tools/Tool/Tool.spec.js

```diff
@@ -84,6 +84,7 @@ describe('Model > DrawingTools > Tool', function () {
       const textTaskSnapshot = Object.assign({}, tool.details[2], {taskKey: 'text'})
       const multipleTask = tool.createTask(multipleTaskSnapshot)
       const singleTask = tool.createTask(singleTaskSnapshot)
+      // TODO come back to this. Why is this passing?
```

Comment:
This is passing because we set `textTask = tool.createTask(singleTaskSnapshot)` then expect  `textTask` to equal itself. The tests should be checking the task against the snapshots.

At the moment, none of these expectations can ever fail. 😳

Reply:
Fixed in #1504.